docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -859,8 +859,13 @@ because it is likely that the number of elements in an array is unknown. For thi
 processor exists. By specifying the field holding array elements and a processor that
 defines what should happen to each element, array fields can easily be preprocessed.
 
-A processor inside the foreach processor works in a different context, and the only valid top-level
-field is `_value`, which holds the array element value. Under this field other fields may exist.
+A processor inside the foreach processor works in the array element context and puts that in the ingest metadata
```

Comment:
do we want to emphasis the fact that `_ingest._value` is overwritten? so if you had a previous metadata field named `_value` in metadata, it will be lost.